src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/UserTests.kt

```diff
@@ -80,6 +82,18 @@ class UserTests : RepositoryTests<UserRepository>()
     }
 
     @Test
+    fun `can update last logged in`()
+    {
+        given(this::addTestUser).check { repo ->
+            val then = Date(System.currentTimeMillis())
+            repo.updateLastLoggedIn(username)
+            val lastLoggedIn = repo.getUserByUsername(username).lastLoggedIn
+            assertThat(lastLoggedIn).isNotNull()
+            assertThat(lastLoggedIn).isBetween(then, Date(System.currentTimeMillis()))
```

Comment:
This is a really nice way of doing a time-based unit test that I've never seen before :)